### ISSUE TITLE
You can't see more than 2 or 3 paths at the same time

### ISSUE THREAD

**Issue by yashha** [Rostlab/JS16_ProjectC_Group10#230]
I have been informed that there is that bug

**Comment by bandle** [comment]
Can you be more exact with it? For me it works.
![bildschirmfoto 2016-04-05 um 21 16 56](https://cloud.githubusercontent.com/assets/11232638/14294941/cf809d20-fb73-11e5-9197-83d4b36dd468.png)

**Comment by bandle** [comment]
I just looked at your page and in fact it doesn't work there. I don't know what you did to break it :/

**Comment by sacdallago** [comment]
:( @AlexMoroz ? :joy:

**Comment by bandle** [comment]
@sacdallago was right - in our build the problem occurs. So when @AlexMoroz just rebundles it, it will hopefully work.

**Comment by sacdallago** [comment]
:dancer: sometimes I am right? This is amazing!

**Comment by bandle** [comment]
I closed it to early :/ Problem still there

**Comment by AlexMoroz** [comment]
What pages do I need to compare?
Where it doesn't work? map.got.show?

**Comment by bandle** [comment]
It only doesn't work in the bundled version :/

Problem is the function which combines multiple marker into one when more than one character is at a place at the same time

**Comment by AlexMoroz** [comment]
> Problem still there

@mbandle  where? =))

**Comment by bandle** [comment]
When for example adding two of the starks - They both start in winter fell. This leads to a problem only in the bundle.

**Comment by bandle** [comment]
Found the problem - Leaflet tries to automatically search for the imagePath, but bundling it breaks this function. I'll add a fix soon.

**Comment by AlexMoroz** [comment]
@mbandle  you are the best :+1: :satisfied:

**Comment by bandle** [comment]
I added the Leaflet marker image from the cdn

```
    L.Icon.Default.imagePath = "http://cdn.leafletjs.com/leaflet/v0.7.7/images";
```

This should work after @AlexMoroz rebounded it

**Comment by AlexMoroz** [comment]
You can just change the npm version and it will be rebundled automatically)

**Comment by AlexMoroz** [comment]
Done. New version is online. Please check.

**Comment by bandle** [comment]
Have you rebundled it, I don't see any file changed except the version number

**Comment by AlexMoroz** [comment]
https://travis-ci.org/Rostlab/JS16_ProjectC_Group10/builds/120990401
yes, it's all done automatically.

**Comment by yashha** [comment]
I get now 
`bundle.js:31 XHR finished loading: GET "https://api.got.show/api/https://api.got.show/api//characters".`
probably an error with our config?
https://github.com/Rostlab/JS16_ProjectF/blob/develop/app/components/common/MapComp/MapComp.jsx#L72

**Comment by AlexMoroz** [comment]
probably, the config was not changed

**Comment by yashha** [comment]
I got it :)
Edit: something changed for sure
Edit: https://github.com/Rostlab/JS16_ProjectF/pull/284/commits/5ca0a9717f8bbfc5f6da08faf43b3d4b4b9b2adc

**Comment by yashha** [comment]
I still get 
`deploy.bundle.js:5 Uncaught Error: Couldn't autodetect L.Icon.Default.imagePath, set it manually.`
at the 4th path  and it deletes all paths

**Comment by bandle** [comment]
But why does travis say this? @AlexMoroz 
![bildschirmfoto 2016-04-06 um 09 11 29](https://cloud.githubusercontent.com/assets/11232638/14308631/9743d40e-fbd7-11e5-8ca2-4adcd84fd3fd.png)

**Comment by AlexMoroz** [comment]
this is usual error because the changes was not committed before publishing to npm
oh, it doesn't publish the files with error, I didn't expect that =(
I rebundled it and republished on my own

**Comment by bandle** [comment]
Works now 
Thanks @AlexMoroz for rebundling

**Comment by AlexMoroz** [comment]
I will try to fix the error with publishing

**Comment by AlexMoroz** [comment]
Error with publishing fixed by #238

**Comment by sacdallago** [comment]
Super!
